src/ts/components/seekbar.ts

```diff
@@ -111,10 +125,17 @@ export class SeekBar extends Component<SeekBarConfig> {
   constructor(config: SeekBarConfig = {}) {
     super(config);
 
+    const keyStepIncrements = this.config.keyStepIncrements != null ? this.config.keyStepIncrements : {
```

Comment:
` != null` is unnecessary. Actually you can do `this.config.keyStepIncrements || { ...`. best would be `??` but that needs TS 3.7.